ringtest.py

```diff
@@ -24,6 +24,12 @@
 # stop time of simulation
 tstop = args.tstop
 
+# Which integration method to use
+method = args.method
+
+# Cvode threshold detection --- first or second order
+condition_order = 2 if args.thresh_order2 is True else 1
```

Comment:
Nit: we don't need the `is True` part since `thresh_order2` is a boolean anyway.

Reply:
fixed